bundles/org.openhab.binding.upb/README.md

```diff
@@ -0,0 +1,101 @@
+# UPB Binding
+
+Universal Powerline Bus (UPB) is a protocol for communication over household electrical wiring.
+
+## Supported Things
+
+The binding has not yet been tested with a variety of devices, so specific device support is limited.
+Instead the binding provides some generic devices, and allows adding channels that match the type of device.
+
+* `serial-pim` - Serial PIM
+* `generic` - Generic UPB device
+* `virtual` - "Virtual" device that allows scene selection
+
+Specific devices that are supported:
+
+ * `leviton-38a00-1` - 6-button scene switch
+
+## Binding Configuration
+
+The following binding configuration parameters are supported:
+
+| Parameter                | Description                                    | Config   | Default |
+| ------------------------ | ---------------------------------------------- |--------- | ------- |
+| networkId                | Default UPB network ID (0-255)                 | Optional | -       |
+
+## Thing Configuration
+
+### Serial PIM
+
+You need a Powerline Interface Module (PIM) for the binding to work. This is a piece of equipment
```

Comment:
Please put each sentences on a single line. that makes it easier to review differences.